custom/icds_reports/management/commands/sql_scripts/ap_data_pull_2.sql

```diff
@@ -0,0 +1,83 @@
+COPY(SELECT
```

Comment:
this query would needs to prepared like this. Else it would give multiple rows per awc


```
COPY(SELECT
    awc.state_name,
    awc.district_name,
    awc.block_name,
    awc.supervisor_name,
    awc.awc_name,
    awc.awc_site_code,
    SUM(CASE WHEN agg.num_awcs_conducted_vhnd>0 AND agg.month='2019-04-01' THEN 1 ELSE 0 END) as vhnd_conducted_April_2019,
    SUM(CASE WHEN agg.num_launched_awcs=1 AND agg.month='2019-04-01' THEN 1 ELSE 0 END) as     launched_status_April_2019,                                                                                       
    SUM(CASE WHEN agg.num_awcs_conducted_vhnd>0 AND agg.month='2019-05-01' THEN 1 ELSE 0 END) as vhnd_conducted_may_2019,
    SUM(CASE WHEN agg.num_launched_awcs=1 AND agg.month='2019-05-01' THEN 1 ELSE 0 END) as     launched_status_may_2019,  
    SUM(CASE WHEN agg.num_awcs_conducted_vhnd>0 AND agg.month='2019-06-01' THEN 1 ELSE 0 END) as vhnd_conducted_June_2019,
    SUM(CASE WHEN agg.num_launched_awcs=1 AND agg.month='2019-06-01' THEN 1 ELSE 0 END) as     launched_status_June_2019
FROM
    "awc_location_local" awc LEFT JOIN "agg_awc" agg ON (
        awc.supervisor_id=agg.supervisor_id and
        awc.doc_id = agg.awc_id AND
        awc.supervisor_id=agg.supervisor_id AND
        agg.month in ('2019-04-01','2019-05-01','2019-06-01')
    )
    where awc.state_id='f98e91aa003accb7b849a0f18ebd7039' and awc.aggregation_level=5
    group by awc.state_name,
    awc.district_name,
    awc.block_name,
    awc.supervisor_name,
    awc.awc_name,
    awc.awc_site_code
    
    
    ) TO '/tmp/ap_data_pull_2.csv' DELIMITER ',' CSV HEADER;

```


this 1/0 can be formated during consolidation

Reply:
updated